tests/benchmarks/timing_info/testcase.yaml

```diff
@@ -1,4 +1,9 @@
 tests:
-  benchmark.timing:
+  benchmark.timing.default_kernel:
     arch_whitelist: x86 arm
     tags: benchmark
+  benchmark.timing.userspace:
+    filter: CONFIG_ARCH_HAS_USERSPACE
+    extra_args: CONF_FILE=prj_userspace.conf
+    arch_whitelist: x86 arm
```

Comment:
how hard would it be to add ARC support?
just need to add another ifdef block for the timer right?

Reply:
currently adding the support. i will be creating another PR once that is done.